tests/data_files/Makefile

```diff
@@ -1345,6 +1414,8 @@ server2.req.sha256: server2.key
 	$(MBEDTLS_CERT_REQ) output_file=$@ filename=$< subject_name="C=NL,O=PolarSSL,CN=localhost" md=SHA256
 all_intermediate += server2.req.sha256
 
+all_parse_input += parse_input/server2.crt.der
+all_parse_input += parse_input/server2.crt
```

Comment:
To be consistent with the rest of the times you appended to all_parse_input, can we move this one below?

Reply:
I will postpone this feedback.

I add those with a script. But this line is added before I use the script. I might update it when preceding PRs were merged.